Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -70,6 +70,20 @@ class DisconnectSetting: WithAccountSetting {
                 self.settings.profile.removeAccount()
                 self.settings.settings = self.settings.generateSettings()
                 self.settings.SELfirefoxAccountDidChange()
+                let loginsController = UIAlertController(
+                    title: NSLocalizedString("Remove Logins?", comment: "Title of the 'Remove firefox logins' alert"),
```

Comment:
Yes, please! For new strings, please use `NSLocalizedString` with the `value` parameter, where the first unnamed parameter is a unique, namespaced ID, and the `value` parameter is the string itself. [Here's one example.](https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Strings.swift#L21)